library/src/main/java/io/ona/kujaku/services/options/TrackingServiceHighAccuracyOptions.java

```diff
@@ -0,0 +1,38 @@
+package io.ona.kujaku.services.options;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+/**
+ * Options for Tracking Service . High Accuracy as we are asking the maximum of available locations from the GPS : gpsMinDistance = 0
+ *
+ * Created by Emmanuel Otin - eo@novel-t.ch 03/07/19.
+ */
+public class TrackingServiceHighAccuracyOptions extends TrackingServiceOptions {
+
+    public TrackingServiceHighAccuracyOptions() {
+        super();
+        this.minDistance = 5;
+        this.gpsMinDistance = 0;
+        this.toleranceIntervalDistance = 1;
+        this.distanceFromDeparture = 10;
+        this.minAccuracy = 50;
```

Comment:
Isn't an accuracy of 50 metres too inaccurate for GPS? What do you think and why is this here?

Reply:
We can change this setting to 20 meters. With this accuracy, the user need to wait a bit longer to get the first tack from the GPS

Reply:
I think when working with GPS, we expect even the first points to have a pretty good accuracy. If it were FusedLocationProvider then 50 metres would be OK. What do you think?

Reply:
The TrackingServiceOptions class is abstract and can be extended with custom parameters.